### PR TITLE
[Google Blockly] CT-193: Code to support Sprite Lab migration to Google Blockly

### PR DESCRIPTION
**Pull request by ebeastlake**
## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/test/helpers/levels_helper_test.rb [hunk 1]
```diff
@@ -360,22 +366,23 @@ def request
...
+  test 'use_google_blockly is true if Experiment is enabled for google_blockly and it is not Sprite Lab start_mode' do
+    @is_start_mode = true
     Experiment.stubs(:enabled?).returns(true)
     @level = build :level
     assert use_google_blockly
     Experiment.unstub(:enabled?)
   end
 
-  test 'use_google_blockly is true if Experiment is enabled for google_blockly' do
+  test 'use_google_blockly is true if Experiment is enabled for google_blockly otherwise' do
```
Comment (molly-moen) [inline dashboard/test/helpers/levels_helper_test.rb]: should `otherwise` be here?

**Review by molly-moen — APPROVED** [review]
Nice!

File: dashboard/test/helpers/levels_helper_test.rb (hunk 1 above)
Comment (ebeastlake) [reply dashboard/test/helpers/levels_helper_test.rb]: Nope! Resolved by merging staging into this branch.

File: dashboard/test/models/levels/gamelab_jr_test.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    DCDO.stubs(:get).with('sprite_lab_google_blockly', false).returns(true)
+    assert @gamelab_jr.uses_google_blockly?
+  end
+
+  test 'uses_google_blockly? returns false when DCDO flag is false' do
+    DCDO.stubs(:get).with('sprite_lab_google_blockly', false).returns(false)
+    refute @gamelab_jr.uses_google_blockly?
+  end
+
+  test 'uses_google_blockly? returns false as default' do
```
Comment (ebeastlake) [inline dashboard/test/models/levels/gamelab_jr_test.rb]: This test will break when we flip the test environment, right? Is that brittle or intended? 🙃 Thoughts @molly-moen?

File: dashboard/test/helpers/levels_helper_test.rb [hunk 3]
```diff
@@ -376,16 +376,6 @@ def request
     reset_view_options
   end
 
-  test 'use_google_blockly is true if blocklyVersion is set to Google in view_options even if start_mode is true' do
```
Comment (ebeastlake) [inline dashboard/test/helpers/levels_helper_test.rb:379]: Now that we're not treating start mode differently, this test tests an edge case that isn't relevant, I think.

File: dashboard/test/models/levels/gamelab_jr_test.rb (hunk 2 above)
Comment (molly-moen) [reply dashboard/test/models/levels/gamelab_jr_test.rb]: hmm yeah I think we probably don't need this one given we have the above two. Because this will behave differently on drone vs test so there's no way to make this work once we flip on test.

Comment (ebeastlake) [reply dashboard/test/models/levels/gamelab_jr_test.rb]: That's what I was worried about, thanks!

**Review by molly-moen — APPROVED** [review]
🎉
